src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -127,7 +126,7 @@ internal abstract partial class DefinitionItem
             ImmutableArray<TaggedText> originationParts,
             ImmutableArray<DocumentSpan> sourceSpans,
             ImmutableDictionary<string, string> properties,
-            ImmutableArray<FindUsageProperty> displayableProperties,
+            ImmutableDictionary<string, string> displayableProperties,
```

Comment:
should this be a multi-dictionary?  in the case of something like 'reference kind' i thought we could have multiple values associated with the key.

Reply:
We don't have a reference kind for `DefinitionItem`. `SymbolUsageInfo` is now a directly property on `SourceReferenceItem`

Reply:
let me rephrase:  we already have the concept of "key => multiple vlaues".  so i would have expected the displayableProperties to follow that as well, and not be a `string => string`.

Reply:
> we already have the concept of "key => multiple vlaues".

Where do we have that concept? We used to have it in SourceReferenceItem, but I have removed that as part of this PR. Now the extensible custom properties for SourceReferenceItem are `string => string`, with an explicit value stored for SymbolUsageInfo. That helped cleanup and simplify bunch of code as we don't need `key => multiple vlaues` anymore.

Reply:
sorry, i'm not bein clear :)

Say i'm a lanuage plugging into FindRefs.  And i want to add a custom column.  *And i would like it to behave like the 'kind' column*.  There appears to be no way for me to do that.  Yes, i could add a column with singular string vlues.  But i could not add a column with multi-values.

This effectively means that hte extensibility system is less capable than what's built in.  :)

Reply:
Sure, that was our goal originally, but none of the concrete features require that currently and was causing unnecessary complexity in code. Once we have an actual customer requesting multivalue scenario, we can refactor the code to support it.

Reply:
i'm ok with that :)    This was more about me thinkin about this as if it was a public APi.  in that case, my preference would be to support this from the start.  But since this is all internal, i think this is 100% fine for now :)

Reply:
One tricky thing with multiple column values is how to handle resource strings to display when there are different combinations of values based on column filtering. Default way of comma based join might not always suffice. Best to avoid adding this support until we have a real scenario that can be tested.

Reply:
My ideal would be that hte runtime would allow you to specify an `object[]` (or `string[]`) and they'd just take care of it.  But i agree there's no pressing need to figure this out currently :)